twindle-cli/src/utils/image.test.js

```diff
@@ -0,0 +1,13 @@
+const { encodeImage } = require("./image");
+
+describe("using mock url to test encoding function", () => {
+  test("matches email", async () => {
```

Comment:
Matches email in image encoding module? 😅

Reply:
o boi! that was a typo